core/CHANGELOG.md

```diff
@@ -2,3 +2,20 @@
 
 * Renamed ItemAdjustments to Adjustable::AdjustmentsUpdater.
 * Removed allow_ssl_in_* preferences from spree
+* Enable configuration of `Carmen.i18n_backend.locale` at `seeds.rb`
```

Comment:
Can add to notable changes of https://github.com/spree/spree/blob/master/guides/content/release_notes/3_0_0.md